app/javascript/dashboard/routes/dashboard/inbox/components/InboxCard.vue

```diff
@@ -131,6 +140,12 @@ export default {
         },
       ];
 
+      if (this.isNotSnoozed) {
```

Comment:
Let us discuss this. I'm not sure about managing this in the view.

Reply:
Sure

Reply:
@pranavrajs I have made some changes here, But in a sperate PR for easy review https://github.com/chatwoot/chatwoot/pull/8924